src/Lean/Data/Lsp/Basic.lean

```diff
@@ -78,6 +78,18 @@ structure Command where
   arguments? : Option (Array Json) := none
   deriving ToJson, FromJson
 
+/-- A snippet string is a template which allows to insert text
+and to control the editor cursor when insertion happens.
+
+A snippet can define tab stops and placeholders with `$1`, `$2`
```

Comment:
I wouldn't call these "tab stops", as that's already used for the multiples-of-spaces that the editor indents to (for Lean code, typically every other column is a tab stop; for Python, it's typically every fourth).

What about just calling them "placeholders"?